BitwardenShared/Core/Vault/Services/Stores/CipherDataStore.swift

```diff
@@ -62,6 +68,13 @@ protocol CipherDataStore: AnyObject {
 }
 
 extension DataStore: CipherDataStore {
+    func cipherCount(userId: String) async throws -> Int {
+        try await backgroundContext.perform {
```

Comment:
is this clock safe to have a strong self because its using the `backgroundContext` within itself?  that might be a dumb question so feel free to ignore me haha

Reply:
This is safe because it's a short-lived closure. The context is going to perform the closure and then immediately return, releasing the self/backgroundContext reference.